lua/lazyvim/plugins/extras/coding/avante.lua

```diff
@@ -0,0 +1,29 @@
+return {
+  {
+    "yetone/avante.nvim",
+    init = function()
+      require("avante_lib").load()
+    end,
```

Comment:
You don't have to do this

Reply:
hi! care to explain why? I wasn't sure by only reading the docs

Reply:
It seems to be needed when you want 
> For avante.tokenizers and templates to work, make sure to call require('avante_lib').load() somewhere when entering the editor. 

Not sure what `avante.tokenizers` and templates are, but it if a user wants to use them they suggest to do what you already do in this PR.

Reply:
It is already loaded in case of lazy.nvim

That doe is only valid for ppl who don't use lazy.nvim

Reply:
https://github.com/yetone/avante.nvim/blob/5c02a5d846a17b9f9db40cf64caa0bc75e7b13c1/lua/avante/init.lua#L236

Reply:
Oh, I see now. What I quoted was under the `Lua` section for when people do plain `require`s in their config without `lazy.nvim`. Ok that makes more sense. 

Thanks for clarifying

Reply:
I updated the docs accordingly sorry for any confusions.